.github/workflows/1x-integration-test.yaml

```
@@ -1,8 +1,37 @@
-name: Test
+name: 1.x Integration Tests
 
-on: [push]
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'next'
+  push:
+    branches:
+      - '*'
+    tags:
+      - '1.*.*'
```

```suggestion
      - '1.*'
      - 'v1.*'
```

Two thoughts:
- Per #1411, our use of tags is not consistent :scream: some are `vX.Y.Z`, some don't have the `v` prefix. So as not to block this PR by that let's just capture both. 
- `1.*` might be more robust than `1.*.*`. The last `*` needs to capture the optional semver suffix (e.g. alpha/beta as well). While both patterns work equally well, the former is shorter and gives a weaker hint as to the exact structure of the tag (which is better in this case, because the last * has to capture the optional suffix as well). This one is optional, use your best judgment.